libs/core/web/src/lib/pages/film-list/FilmList.tsx

```diff
@@ -0,0 +1,33 @@
+import { i18nKeys } from '@form-exercise/i18n';
+import Typography from '@mui/material/Typography';
+import Grid from '@mui/material/Grid';
+import { useTranslation } from 'react-i18next';
+import { SmallInfoCard } from '@form-exercise/ui';
+import { useFetchOmdb } from '@form-exercise/utils';
+import { Film } from '@form-exercise/data/interface';
+
+export function FilmList() {
+  const { t } = useTranslation();
+  const { isLoading, data: films } = useFetchOmdb({ searchBy: 'Batman' });
```

Comment:
next step would be to implement a seach field to set the `searchBy` parameter 👀 (bonus point: with autocomplete if the omdb api allows it)